### DISCUSSION TITLE
Great Expectations Operator XCOM Error

### DISCUSSION THREAD

**Discussion by mayankjoshi954** [apache/airflow#29778] [Q&A]
I'm using airflow version 2.2.2 with great expectations version 0.15.50
I created a DAG with GreatExpectationsOperator, the task is getting failed due to this error: 

{xcom.py:334} ERROR - Could not serialize the XCom value into JSON. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config.

The validations through GreatExpecttion operator is successful, but at the end this error appears due to which the task is marked as failed.
PS: I tried changing enable_pickle in airflow.cfg but it did not help.

**Comment by potiuk** [comment]
Airflow had pickle support for XCOM which is disabled by default - look for it in the docs/configuration. But likely this is something the the team managing GE should fix - using pickling for xcoms is not the best idea (stability/security) and it is likely that they should change it. Maybe new version of operator already had it who knows. You need to contact them as GE is operator developed outside of the Apache Airflow community - so likely raise an issue there.

**Reply by josh-fell** [reply]
CC @denimalpaca

**Comment by denimalpaca** [comment]
@mayankjoshi954 you can open the PR in this repo:
https://github.com/astronomer/airflow-provider-great-expectations/issues
Please include the version of the operator you're using as well!

Also look in to [this](https://github.com/astronomer/airflow-provider-great-expectations/blob/main/great_expectations_provider/operators/great_expectations.py#L110) parameter, it helps with serialization issues.
